docs/platforms/java/common/opentelemetry/setup/agentless.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: OpenTelemetry Agentless
+sdk: sentry.java.opentelemetry-agentless
+description: "Using OpenTelemetry with  sentry-opentelemetry-agentless."
+sidebar_order: 200
+---
+
+If you do not want to use our recommended <PlatformLink to="/opentelemetry/setup/agent">Java Agent</PlatformLink>, we also offer a dependency that allows you to use OpenTelemetry with Sentry.
```

Comment:
Should we add an example use case here for when to use the agentless version instead of the agent, like when using GraalVM?
Could also be done in https://github.com/getsentry/sentry-java/issues/4313